client/my-sites/stats/stats-subscribers-overview/index.tsx

```diff
@@ -1,57 +1,113 @@
 import { CountComparisonCard } from '@automattic/components';
+import { UseQueryResult, useQueries } from '@tanstack/react-query';
 import { translate } from 'i18n-calypso';
 import React from 'react';
+import {
+	querySubscribers,
+	selectSubscribers,
+} from 'calypso/my-sites/stats/hooks/use-subscribers-query';
+
+const indexFirstCard = 0; // 0 for today
```

Comment:
1/3 I can see when the days are used, they are used as an array - I'm just wondering, would it be better to use an array here to avoid so many variables.